fideslib/oauth/schemas/oauth.py

```diff
@@ -61,35 +37,36 @@ def __init__(
     ):
         if not scopes:
             scopes = {}
-
         flows = OAuthFlows(clientCredentials={"tokenUrl": tokenUrl, "scopes": scopes})
         super().__init__(
-            auto_error=auto_error,
-            description=description,
             flows=flows,
             scheme_name=scheme_name,
+            description=description,
+            auto_error=auto_error,
         )
 
     async def __call__(self, request: Request) -> Optional[str]:
-        scheme, param = get_authorization_scheme_param(
-            request.headers.get(key="Authorization", default="")
-        )
-
-        if scheme.lower() != "bearer":
+        authorization: str = request.headers.get("Authorization")
+        scheme, param = get_authorization_scheme_param(authorization)
+        if not authorization or scheme.lower() != "bearer":
             if self.auto_error:
-                raise InvalidAuthorizationSchemeException()
-
+                raise HTTPException(
+                    status_code=HTTP_401_UNAUTHORIZED,
+                    detail="Not authenticated",
+                    headers={"WWW-Authenticate": "Bearer"},
+                )
```

Comment:
Why remove the custom exceptions?

Reply:
It wasn't used anywhere.

Reply:
I meant the exceptions previously defined in https://github.com/ethyca/fideslib/blob/main/fideslib/oauth/api/exceptions.py - they were basically the same, just more organized and intuitive to call. Still, I think their names could stand to be updated to end in `...Error` instead of `...Exception`, to align with convention.